presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentListener.java

```diff
@@ -41,6 +41,8 @@
 {
     Logger log = Logger.get(EnvironmentListener.class);
 
+    EnvironmentListener NO_OP = new EnvironmentListener() {};
```

Comment:
too acronymish IMO. `EMPTY`?

Reply:
EMPTY is more suitable for values, not action objects. We use NOOP in some places